src/Illuminate/Contracts/Routing/LocaleManager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Illuminate\Contracts\Routing;
+
+/**
+ * Interface LocaleManager
+ *
+ * @package Illuminate\Contracts\Routing
```

Comment:
Please never use the package annotation. It's only ever meant to be used in php 5.2 code where you can't have actual namespaces.